js/src/forum/index.ts

```diff
@@ -131,7 +112,7 @@ app.initializers.add('flarum-pusher', () => {
             if (!document.hasFocus()) {
               app.setTitleCount(Math.max(0, this.discussion.commentCount() - oldCount));
 
-              $(window).one('focus', () => app.setTitleCount(0));
+              window.addEventListener('focus', () => app.setTitleCount(0));
```

Comment:
Need to add the `once` option to this to keep the behavior the same.  `one` makes the callback be called for the 1st event per element, and here it's the same as once.